data.py

```diff
@@ -29,19 +29,42 @@ def __call__(self, samples):
 
         return all_texts, texts_mask, all_strokes, strokes_mask
 
-class HandwritingDataset(Dataset):
+class Data:
```

Comment:
I think that this is ok! but personally, I think that I'd just do this in the dataset class directly. Since you're exposing the methods that we'll need later in the dataset class, what do you think are the advantages of keeping a separate Data class?

Reply:
I wasn't sure about passing in the dataset to the model.  Have done so now, though.